drivers/peci/peci_mchp_xec.c

```diff
@@ -296,8 +318,9 @@ static int peci_xec_transfer(const struct device *dev, struct peci_msg *msg)
 
 	/* If a PECI transmission is successful, it may or not involve
 	 * a read operation, check if transaction expects a response
+	 * Also perform a read when PECI cmd is Pingt to get Write FCS
```

Comment:
Is `Pingt` the actual command name?

Reply:
@dcpleung no, this is a typo. I will rework it